dts/common/nordic/nrf54l09.dtsi

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "nrf54l_05_10_15.dtsi"
+
+/delete-node/ &gpio2;
+/delete-node/ &spi00;
+/delete-node/ &uart00;
+/delete-node/ &spi22;
+/delete-node/ &i2c22;
+/delete-node/ &uart22;
+/delete-node/ &pdm20;
+/delete-node/ &pdm21;
+/delete-node/ &pwm20;
+/delete-node/ &pwm21;
+/delete-node/ &pwm22;
+/delete-node/ &i2s20;
+/delete-node/ &qdec20;
+/delete-node/ &qdec21;
+/delete-node/ &nfct;
+
+&cpuapp_sram {
+	reg = <0x20000000 DT_SIZE_K(144)>;
+	ranges = <0x0 0x20000000 DT_SIZE_K(144)>;
+};
+
+/* 144 + 48 = 192KB */
+/ {
+	soc {
+		cpuflpr_sram: memory@20024000 {
+			compatible = "mmio-sram";
+			reg = <0x20024000 DT_SIZE_K(48)>;
+			#address-cells = <1>;
+			#size-cells = <1>;
+			ranges = <0x0 0x20024000 DT_SIZE_K(48)>;
+		};
+	};
+	cpus {
+		cpuapp: cpu@0 {
+			clock-frequency = <DT_FREQ_M(64)>;
+		};
+	};
+};
+
+&cpuapp_rram {
+	reg = <0x0 DT_SIZE_K(950)>;
```

Comment:
There is a problem with sizes that are not a multiple of `erase-block-size`.
https://github.com/zephyrproject-rtos/zephyr/blob/bd69ae46bc63821d0eb0955611e4804ed7d14f16/drivers/flash/soc_flash_nrf_rram.c#L52
The number of pages will be rounded-down when using the `nrf_rram_page_layout`.

Reply:
changed to 952 and 60 for flpr.

Reply:
Was this change commited? It seems the value is still at `950`.

Reply:
not sure what happen but should be ok now.